Update Expo documentation (#1251)
microsoft/vscode-react-native: README.md

To debug a project created using Expo or the `create-react-native-app` task, you can use embedded support for Expo.

> Prepare your environment using an [introduction](https://reactnative.dev/docs/environment-setup). Pay attention to:

Reviewer: ```suggestion
Prepare your environment the [Expo CLI Quickstart instruction](https://reactnative.dev/docs/environment-setup). 
For correct work with Expo this extension **`requires Android SDK`**.
So also pay attention to the `React Native CLI Quickstart` tab, where you can find the Android SDK installation guide:
```

- Install the [Expo app](https://getexponent.com/) on the target device or emulator

> - Ensure that the `Android SDK` is installed on your computer (You may install it with [Android Studio](https://developer.android.com/studio))

Reviewer: It is better to point out how exactly it should be installed. For example after installation of Android SDK some of `PATH` environment variables may absent and this might lead to errors as in #1230

Reviewer 2: Maybe suggest too the option to run `npx react-native doctor` to validate that everything is installed and configured correctly?
Soon-ish [doctor will install Android Studio and SDK on Windows automatically 😊](https://github.com/react-native-community/cli/pull/1109)

Reviewer: Sounds good! It may help to find gaps in Android SDK configuration.

To debug a project created using Expo or the `create-react-native-app` task, you can use embedded support for Expo.

> Prepare your environment the [Expo CLI Quickstart instruction](https://reactnative.dev/docs/environment-setup).

Reviewer 3: ```suggestion
Prepare your environment by following the [Expo CLI Quickstart instruction](https://reactnative.dev/docs/environment-setup).
```

For correct work with Expo this extension **`requires Android SDK`**.
So also pay attention to the `React Native CLI Quickstart` tab, where you can find the Android SDK installation guide:

- Install the [Expo app](https://getexponent.com/) on the target device or emulator

> - Ensure that the `Android SDK` is installed on your computer (You may install it with [Android Studio](https://developer.android.com/studio))

Reviewer: ```suggestion
- Ensure that the `Android SDK` is installed on your computer (You may install using the part of [`React Native CLI Quickstart` guide](https://reactnative.dev/docs/environment-setup))
```

> - Ensure that the `Android SDK` is installed on your computer (You may install using the part of [`React Native CLI Quickstart` guide](https://reactnative.dev/docs/environment-setup))

Reviewer: ```suggestion
- Ensure that the `Android SDK` is installed on your computer (You may install it using the [`React Native CLI Quickstart` guide](https://reactnative.dev/docs/environment-setup))
```

Your environment must meet the following prerequisites:

- Install the [Expo app](https://getexponent.com/) on the target device or emulator
- Ensure that the `expo-cli` is installed globally (`npm install -g expo-cli`)
- [More details](https://reactnative.dev/docs/environment-setup)

To start debugging in Expo follow these steps:

1. Open your project in VS Code with this extension installed.
1. Create a debug configuration (as described in [Debugging React Native applications](#debugging-react-native-applications)), select `Debug in Exponent` in the debug drop-down menu, and start debugging

> 1. Wait while some dependencies are configured - the extension will install `xdl` and `@expo/ngrok` when this feature is used for the first time.

Reviewer: ```suggestion
1. Wait while some dependencies are configured - the extension will install [`Expo Development Library(@expo/xdl)`](https://www.npmjs.com/package/@expo/xdl)  when this feature is used for the first time.
```
